code/modules/client/preference_setup/loadout/loadout.dm

```diff
@@ -432,24 +434,34 @@ var/list/gear_datums = list()
 		return "You cannot spawn with the [initial(spawning_item.name)] with your current origin!"
 	return null
 
-/datum/gear/proc/spawn_item(var/location, var/metadata, var/mob/living/carbon/human/H)
+/// Returns a list of spawn item data. The first entry is the path, the second is the location
+/datum/gear/proc/get_spawn_item_data(var/location, var/metadata, var/mob/living/carbon/human/H)
 	var/datum/gear_data/gd = new(path, location, faction)
 	for(var/datum/gear_tweak/gt in gear_tweaks)
 		if(metadata["[gt]"])
 			gt.tweak_gear_data(metadata["[gt]"], gd, H)
 		else
 			gt.tweak_gear_data(gt.get_default(), gd, H)
-	if(ispath(gd.path, /obj/item/organ/external))
-		var/obj/item/organ/external/external_aug = gd.path
+	return list(gd.path, gd.location)
+
+/datum/gear/proc/spawn_item(var/location, var/metadata, var/mob/living/carbon/human/H)
```

Comment:
Probably should be marked as SHOULD_NOT_SLEEP(TRUE)

Reply:
tried adding it and it exploded the linter, looks like changes unrelated to this PR so im not gonna bother